action.yml

```diff
@@ -34,50 +34,52 @@ runs:
       shell: bash
       run: |
         VERSION=""
-        IS_RELEASE=false
+        IS_RELEASE=true
 
-        # Select only those releases we wish to allow, with or without pre-releases
-
-        if [[ -z "${{ inputs.gap-version }}" || "${{ inputs.gap-version }}" == "latest" ]] ; then
-          echo "Version: latest release"
-          IS_RELEASE=true
-          VERSION=$(wget --header="Authorization: Bearer ${{ github.token }}" -qO- "https://api.github.com/repos/${{ inputs.repository }}/releases/latest" | jq -r '.tag_name')
-        elif [[ "${{ inputs.gap-version }}" =~ ^(master|main|default)$ ]] ; then
-          echo "Version: default branch"
-          VERSION=$(git ls-remote --symref https://github.com/${{ inputs.repository }}.git HEAD | head -n 1 | sed 's|ref: refs/heads/||; s|\tHEAD||')
-        else
-          echo "Checking releases"
-          GIT_RELS_JSON=$(wget --header="Authorization: Bearer ${{ github.token }}" -qO- "https://api.github.com/repos/${{ inputs.repository }}/releases")
-          GIT_RELS=$(echo "$GIT_RELS_JSON" | jq -r '.[].tag_name' | sort -V)
-          GIT_NPRS=$(echo "$GIT_RELS_JSON" | jq -r '.[] | select((.draft == false) and (.prerelease == false)) | .tag_name' | sort -V)
-
-          # Add "v" in front if missing
-          REL=${{ inputs.gap-version }}
-          REL=v${REL#v}
-          
-          if echo "$GIT_RELS" | grep -qx "$REL" ; then
-            echo "Version: exact release match"
-            IS_RELEASE=true
-            VERSION=$REL
-          elif echo "$GIT_NPRS" | grep -q "^$REL\." ; then
-            echo "Version: expanded to release"
-            IS_RELEASE=true
-            VERSION=$(echo "$GIT_NPRS" | grep "^$REL\." | tail -n 1)
+        # We only check for releases in the official repository
+        if [[ "${{ inputs.repository }}" == "gap-system/gap" ]] ; then
+          wget -q -O $RUNNER_TEMP/gap_releases.json "https://raw.githubusercontent.com/gap-system/GapWWW/refs/heads/master/_data/gap_releases.json" 
```

Comment:
TODO: change the URL to `https://gap-system.org/releases.json` once https://github.com/gap-system/GapWWW/pull/389 is merged.